extensions/big-data-cluster/README.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft SQL Server Big Data Cluster Extension for Azure Data Studio
+
+Welcome to Microsoft SQL Server Big Data Cluster Extension for Azure Data Studio!
```

Comment:
@kevcunnane should BDC be capitalized here, or not?

Reply:
@chlafreniere , it should not as Kevin pointed our in another comment, I will update it.